libs/application/templates/parental-leave/src/forms/Prerequisites.ts

```diff
@@ -26,6 +29,65 @@ export const PrerequisitesForm: Form = buildForm({
       id: 'prerequisites',
       title: parentalLeaveFormMessages.shared.prerequisitesSection,
       children: [
+        buildSubSection({
+          id: 'mockData',
+          title: 'Gervigögn',
+          children: [
+            buildMultiField({
+              id: 'shouldMock',
+              title: 'Gervigögn',
+              children: [
+                buildRadioField({
+                  id: 'useMockData',
+                  title: 'Viltu nota gervigögn?',
+                  options: [
+                    {
+                      value: YES,
+                      label: 'Já',
+                    },
+                    {
+                      value: NO,
+                      label: 'Nei',
+                    },
+                  ],
+                }),
+                buildRadioField({
+                  id: 'useMockedParentalRelation',
+                  title: 'Tengsl við barn:',
+                  condition: (answers) => answers.useMockData === YES,
+                  options: [
+                    {
+                      value: 'primary',
+                      label: 'Móðir',
+                    },
+                    {
+                      value: 'secondary',
+                      label: 'Hitt foreldri',
+                    },
+                  ],
+                }),
+                buildTextField({
+                  id: 'useMockedDateOfBirth',
+                  title: 'Áætlaður fæðingardagur:',
+                  condition: (answers) =>
+                    answers.useMockData === YES &&
+                    !!answers.useMockedParentalRelation,
+                  placeholder: 'YYYY-MM-DD',
+                  format: '####-##-##',
+                }),
+                buildTextField({
+                  id: 'useMockedPrimaryParentNationalRegistryId',
+                  title: 'Kennitala móður:',
+                  condition: (answers) =>
+                    answers.useMockData === YES &&
+                    answers.useMockedParentalRelation === 'secondary',
+                  placeholder: '1234567-7890',
+                  format: '######-####',
+                }),
+              ],
+            }),
+          ],
+        }),
```

Comment:
I intentionally skipped translations for this, as it is temporary™.

Do we need it?

Reply:
Agree, it's only temporary™. However, I would probably add a condition on the whole section to not show up on production, just in case™.